app/controllers/medical_records_controller.rb

```diff
@@ -22,6 +21,7 @@ def edit
   # POST /medical_records or /medical_records.json
   def create
     @medical_record = MedicalRecord.new(medical_record_params)
+    @medical_record.usuario_id = current_usuario.id
```

Comment:
I think this way is better:
```suggestion
    medical_record_params.merge!(usuario_id: current_usuario.id)
    @medical_record = MedicalRecord.new(medical_record_params)
```

Reply:
Also you can send the current_user_id as part of your form.

Reply:
No, she cannot receive the `usuario_id` from scary internet, it should be set in the backend with the logged in user.

Reply:
You're right.